### PR TITLE
Spanish translation file

### PR DESCRIPTION
**Pull request by JoseDeFreitas**
Hello 👋.
I've just completed the translation of the README file into Spanish.
I made 2 commits. I translated everything in the first one and fixed some typos in the second one.
Here's the Spanish word into Spanish so you can add it to the original README file: `Español`.

### CODE REVIEW

**Comment by anuraghazra** [comment]
Nice, thanks @JoseDeFreitas Can you pleasse link the translation to the main readme file

**Comment by JoseDeFreitas** [comment]
@anuraghazra as you see I just added the link to the Spanish file in the main README file 👍.

File: readme_es.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&theme=radical)
+```
+
+#### Todos los temas incorporados:
+
+dark, radical, merko, gruvbox, tokyonight, onedark, cobalt, synthwave, highcontrast, dracula
+
+<img src="https://res.cloudinary.com/anuraghazra/image/upload/v1595174536/grs-themes_l4ynja.png" alt="Github Readme Stat Themes" width="600px"/>
+
+Descubre más temas en el [archivo de configuración](./themes/index.js). También, **puedes contribuir añadiendo más temas** si lo deseas :D
```
Comment (nombrekeff) [inline readme_es.md:83]: I added a theme preview file earlier, we can add a link to it here.

I would replace that with: 
```diff

- Descubre más temas en el [archivo de configuración](./themes/index.js). También, **puedes contribuir añadiendo más temas** si lo deseas :D
+ Puedes ver una vista previa de [todos los temas disponibles](./themes/README.md) o ver el [archivo de configuración](./themes/index.js) del tema y también **puedes contribuir con nuevos temas** si lo deseas: D 
```

**Comment by nombrekeff** [comment]
Apart from what I mentioned above, it looks all good to me!

File: readme_es.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  </p>
+</p>
+<p align="center">¿Te gusta este proyecto? ¡Por favor considera <a href="https://www.paypal.me/anuraghazra">donar</a> para ayudar a mejorarlo!
+
+# Características
+
+- [Tarjeta de estadísticas de GitHub](#tarjeta-de-estadísticas-de-github)
+- [Pins extra de GitHub](#pins-extra-de-github)
+- [Temas](#temas)
+- [Customización](#customización)
```
Comment (nombrekeff) [inline readme_es.md:34]: @JoseDeFreitas Another thing I would change would be "customizar" for "personalizar", wherever it appears.

**Comment by anuraghazra** [comment]
Great work @JoseDeFreitas 

Also thanks for review @nombrekeff 

Maybe @nombrekeff can update the spanish translation every week, so it's always updated :)

**Comment by nombrekeff** [comment]
Yup, I can try to update it every week, and maybe @JoseDeFreitas can help out if whenever I can't!

**Pull request merged**